src/chrome/content/rules/Bitcoin.pl.xml

```diff
@@ -1,4 +1,9 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://bitcoin.pl/login => https://bitcoin.pl/login: None
+Non-2xx HTTP code: http://www.bitcoin.pl/ (200) => https://www.bitcoin.pl/ (303)
+
```

Comment:
The problem here was that the checker didn't understand 303 as a redirect status. Fixed.